atlp-rwanda/Phantom-frontend-pirates
# ch-setup CI & Quality Assurance pipeline

## PR introduction
### This chore is for setting up Continuous Integration & Quality Assurance pipeline 

## Description 

1. Configure a continuous integration pipeline using GitHub Actions and Heroku. 
2. The continuous integration service should automatically run tests and other preliminary build steps required to ship the next iteration of the product
3. Configure the testing environment with jest or testing library to ensure that all functionalities are tested before deployment

## How should this be manually tested 
1. Clone this repo and cd into this branch
2. Install all the packages used by running `npm install`
4. Run the `npm run test` command to run the test and verify if there are already configured.
